gateway-contracts/contracts/Decryption.sol

```diff
@@ -95,70 +105,109 @@ contract Decryption is
         bytes extraData;
     }
 
-    /// @notice The publicKey and ctHandles from user decryption requests used for validations during responses.
+    /**
+     * @notice The publicKey and ctHandles from user decryption requests used for validations during responses.
+     */
     struct UserDecryptionPayload {
         /// @notice The user's public key to be used for reencryption.
         bytes publicKey;
         /// @notice The handles of the ciphertexts requested for a user decryption
         bytes32[] ctHandles;
     }
 
-    /// @notice The address of the GatewayConfig contract for checking if a signer is valid
+    /**
+     * @notice The address of the GatewayConfig contract for checking if a signer is valid
+     */
     IGatewayConfig private constant GATEWAY_CONFIG = IGatewayConfig(gatewayConfigAddress);
 
-    /// @notice The address of the MultichainACL contract for checking if a decryption requests are allowed
+    /**
+     * @notice The address of the MultichainACL contract for checking if a decryption requests are allowed
```

Comment:
nit: maybe we can also add punctuation at the end of some @notice, like here and line 119 above.